packages/integrations/node/src/response-iterator.ts

```diff
@@ -35,8 +34,8 @@ function isBuffer(value: any): value is Buffer {
 	);
 }
 
-function isNodeResponse(value: any): value is NodeResponse {
-	return !!(value as NodeResponse).body;
+function isNodeResponse(value: any): value is Response {
+	return !!(value as Response).body;
 }
```

Comment:
Do we even need this util anymore?

Reply:
I think it's used to check the types below, and could probably be `instanceof Response` too. But I think I'll leave it for now to simplify the change.